### ISSUE TITLE
project missing license files

### ISSUE THREAD

**Issue by ImUrX** [jkcoxson/netmuxd#40]
Could this have a license file added so it can be packaged on distros?

**Comment by jkcoxson** [comment]
Absolutely! I'm not too familiar with different lincenses, but I'll take a look at it.
